metric/sqs_test.go

```diff
@@ -0,0 +1,61 @@
+package metric
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/aws/aws-sdk-go/service/sqs/sqsiface"
+)
+
+type mockedQueueAttributes struct {
+	sqsiface.SQSAPI
+	Resp sqs.GetQueueAttributesOutput
+}
+
+type sqsTest struct {
+	Resp     sqs.GetQueueAttributesOutput
+	Expected int
+}
+
+// Mock SQS API call and just return the response that is parsed
+func (m mockedQueueAttributes) GetQueueAttributes(in *sqs.GetQueueAttributesInput) (*sqs.GetQueueAttributesOutput, error) {
+	return &m.Resp, nil
+}
+
+func TestUpdateCurrent(t *testing.T) {
+	queueURL := "https://sqs.us-east-1.amazonaws.com/1234567890/microscaling-test"
+
+	cases := make([]sqsTest, 2)
```

Comment:
This is fine, but I think it's more idiomatic to write the cases as a literal (and then you don't have to worry about updating how many there if you add another test case)